src/components/common/GreenhouseEffectSettings.js

```diff
@@ -56,7 +57,7 @@ const GreenhouseEffectSettings = () => {
       <SliderWithLabel
         text={t('Albedo (%)')}
         max={ALBEDO_MAX_VALUE}
-        value={albedo.toFixed(1)}
+        value={+albedo.toFixed(1)}
```

Comment:
I know that this is cool JS, but it might be a bit abstruse for newcomers. Maybe use `Number` or `parseFloat` instead.